aspnet/tutorials/index.rst

```diff
@@ -10,4 +10,5 @@ Tutorials
    publish-to-azure-webapp-using-vs
    nano-server
    ASP.NET 5 and Azure Service Fabric <https://azure.microsoft.com/en-us/documentation/articles/service-fabric-add-a-web-frontend/>
-   
+   Create an ASP.NET 5 web app in Visual Studio Code <https://azure.microsoft.com/documentation/articles/web-sites-create-web-app-using-vscode/>
+   Build a web service front-end for your application <https://azure.microsoft.com/documentation/articles/service-fabric-add-a-web-frontend/>
```

Comment:
Isn't this a duplicate of the one of the links already in the TOC? See line #12.